bm-persona/Libraries/LibraryViewController.swift

```diff
@@ -7,17 +7,47 @@
 //
 
 import UIKit
+import MapKit
 
-class LibraryViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+class LibraryViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, CLLocationManagerDelegate {
+    
+    private static let highRed = UIColor(red: 221/255, green: 67/255, blue: 67/255, alpha: 1)
+    private static let medOrange = UIColor(red: 251/255, green: 179/255, blue: 43/255, alpha: 1)
+    private static let lowGreen = UIColor(red: 162/255, green: 183/255, blue: 14/255, alpha: 1)
+    private static let nearbyDistance: Double = 10
+    private static let invalidDistance: Double = 100
```

Comment:
Instead of displaying an invalid distance for missing user location data, I think we should not display a distance label at all. Same goes for the other details.